recommendations/io-readmore.md

```diff
@@ -13,29 +13,20 @@ The profiled process has been observed is unusually idle under load, typically t
 it's waiting for external I/O because there's nothing else to do until the I/O completes.
 
 To solve I/O issues we have to track down the asynchronous call(s) which are taking an
-abnormally long time to complete.
+abnormally long time to complete. I/O root cause analysis is mostly a reasoning exercise.
 
-I/O root cause analysis is mostly a reasoning exercises, or requires advanced knowledge and
-expertise with specialist Node.js logging flags and (very new) asynchronous tracking API's.
-
-At nearForm we care a lot about this problem and we are developing a new tool to make
-I/O debugging easier... stay tuned!
+It used to requires advanced knowledge and
```

Comment:
I disagree with this approach - I think the text should be more timeless - I don't think it should sound like marketing blurb here. The tone, imo, should reflect that of the event loop recommendation, namely "Bubbleprof is the tool to use in this situation"